core/standard/clause_7_building_blocks.adoc

```diff
@@ -51,9 +51,9 @@ Other encodings are allowed but are not described in this document.
 |===
 |Property |Requirement |Description |GeoJSON key
 |geometry |**required** |A geometry associated with the resource that is used for discovery.  Can be null if there is no associated geometry. |`geometry`
-|time |**required** |The temporal extent of the resource. Can be null if there is no associated temporal extent. |`time`
 |type |**required** |The nature or genre of the resource. |`properties.type`
 |title |**required** |A human-readable name given to the resource. |`properties.title`
+|time |optional |The temporal extent of the resource. Can be null if there is no associated temporal extent. |`time`
```

Comment:
If `time` is now optional, do we care about specifying null rather than not specifying `time` at all?  Suggest the description be updated to only `The temporal extent of the resource.`

Reply:
We do care.  Even though we are making time optional to be compatible with STAC, we should  try not to impede eventual comparability with JSON-FG.  Allowing time to be present and null allows comparability with the JSON-FG time element.

Reply:
We were already directly compatible with JSON-FG.  So I guess this is a middle ground. Should JSON-FG consider making a similar change? This is outside the scope of this PR, of course.

Reply:
@tomkralidis if I am remembering correctly, the last discussion was had about this at the London Code (2023) sprint was that we would NOT add a JSON-FG conformance class to the standard.  So, if we are compatible with JSON-FG (which I actually don't think we are ... but I could be wrong), it would only be by conicidence.  So, yes, this is a middle ground.  As for the JSON-FG SWG considering making the time member optional, that is something for them to consider.